apps/next-docs/content/components/ActionMenu/index.mdx

```diff
@@ -7,6 +7,8 @@ ready: true
 figma: 'https://www.figma.com/file/BJ95AjraesmRCWsKA013GS/Primer-Brand?node-id=4912%3A32799'
 source: https://github.com/primer/brand/tree/main/packages/react/src/ActionMenu/ActionMenu.tsx
 storybook: '/brand/storybook/?path=/story/components-actionmenu--default'
+thumbnail: '/images/thumbnails/action-menu-thumbnail.png'
+thumbnail_darkMode: '/images/thumbnails/action-menu-thumbnail-dark.png'
```

Comment:
Noting for posterity.. the existing frontmatter convention for Doctocat was to use - instead of _ separators. E.g. `menu-position`. We should correct this at some point in future Doctocat release.